e2e/fabricaCommands.test.js

```diff
@@ -49,7 +52,7 @@ describe('validate', () => {
     expect(commandResult).toEqual(TestCommands.success());
     expect(commandResult.output).toContain('Validation errors count: 0');
     expect(commandResult.output).toContain('Validation warnings count: 0');
-    expect(commands.getFiles()).toEqual(['./e2e/__tmp__/commands-tests/fabrica-config.json']);
+    expect(commands.getFiles()).toContain('./e2e/__tmp__/commands-tests/fabrica-config.json');
```

Comment:
hm... didn't quite get the case where:
1. user first creates own chaincode in `chaincodes/chaincode-kv-node`
2. then executes `fabrica init`
  
What would happen then? Files will get overridden or we revive yeoman prompt?  

(If it's not handled) Maybe he should get suitable prompt that chaincode already exists and it's advised to manually check if `chaincode.lang` is correct in fabrica-config ?

Reply:
to be tested